modules/cluster-logging-collector-rsyslog.adoc

```diff
@@ -0,0 +1,174 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-rsyslog_{context}"]
+= Sending logs using the Fluentd *syslog* plug-in (RFC 3164)
+
+You can use the Fluentd *syslog* plug-in to send a copy of its logs to an external syslog server, 
+and not the default Elasticsearch. Note the following about the *syslog* plug-in:
+
+* uses syslog protocol (RFC 3164), not RFC 5424;
+* does not support TLS and thus, is not secure;
+* does not provide Kubernetes metadata, systemd data, or other metadata.
+
+[NOTE]
+====
+This legacy Fluentd *syslog* plug-in method is deprecated and will be removed in a future release.
+====
+
+There are two versions of the Fluentd *syslog* plug-in:
+
+* *out_syslog*. Use `syslog` in the ConfigMap for the non-buffered implementation, communicating through UDP. The non-buffered mode does not buffer data and writes out results
+immediately. 
+* *out_syslog_buffered*: Use `syslog_buffered` in the ConfigMap for the buffered implementation, communicating through TCP. 
+
+To configure the Fluentd *syslog* plug-in, create a ConfigMap called `sysconfig.conf` in the `openshift-logging` namespace. 
+
+[IMPORTANT]
+====
+Changes introduced by the new log forward feature modified the support for Fluentd *syslog* plug-in starting with the {product-title} 4.3 release. In {product-title} 4.3, you create a ConfigMap, as described below, to configure the Fluentd *syslog* plug-in. Any updates to the `secure-forward.conf` section of the Fluentd ConfigMap are removed. Before upgrading cluster logging, you can copy your current `journald.conf` section and use the copied data when you create the `secure-forward` ConfigMap. 
+
+Additionally, you can add any certificates required by your configuration to a secret named `secure-forward` that will be mounted to the Fluentd Pods.
+====
+
+You can use multiple syslog servers by specifying separate `<store>` stanzas in the configuration file.
+
+.Sample `sysconfig.conf`
+[source,yaml]
+----
+<store>
+@type syslog_buffered <1>
+remote_syslog rsyslogserver.openshift-logging.svc.cluster.local <2>
```

Comment:
I suggest using rsyslogserver.example.com instead of rsyslogserver.openshift-logging.svc.cluster.local